homework/insensitive-palindrom/insensitivePalindrom.cpp

```diff
@@ -0,0 +1,13 @@
+#include "insensitivePalindrom.hpp"
+#include <algorithm>
+#include <cctype>
+
+bool is_palindrome(const std::string& palindrome) {
+    std::string removedSigns;
+    std::copy_if(palindrome.begin(), palindrome.end(), std::back_inserter(removedSigns), [](unsigned char ch){ return std::isalnum(ch);});
+    std::transform(removedSigns.begin(), removedSigns.end(), removedSigns.begin(), [](auto& ch){return std::tolower(ch);});
```

Comment:
Use transform and erse -> iterator returned by transform will be last valid, and rest of it you can erase, you will avoid copy_if